.travis.yml

```diff
@@ -26,5 +26,6 @@ before_script:
 install:
   - pip3 install -r requirements.txt
 script:
-  - python -m unittest discover tests -v
+  - coverage run -m unittest discover tests
   - pytest --cov tests
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
There should be a blank line at the end

Reply:
@devkapilbansal, can you please elaborate why we need this blank line? would like to learn the reason behind it 😉.

Reply:
It is as per PEP8 guidelines `W391`. All python files should have a blank line at the end. 
Actually it's not like adding a blank line at the end but terminating the line with a newline character (although both sounds same)